app/src/main/java/org/mozilla/focus/utils/ToastManager.kt

```diff
@@ -21,16 +21,14 @@ object ToastManager {
 
     fun showUnpinnedToast(context: Context) {
         val brandName = context.getString(R.string.firefox_brand_name)
-        context.let {
-            context.getString(R.string.notification_unpinned_general2, brandName)
-        }?.let { string -> showToast(string, context) }
```

Comment:
What was I doing here.... 🤦‍♂️

Reply:
It looks like a Haskell thing, if you ask me. 😝 

I think this got progressively refactored to get more confusing though.